cardano-db-sync/src/Cardano/DbSync/Util/Whitelist.hs

```diff
@@ -45,32 +51,44 @@ isSimplePlutusScriptHashInWhitelist :: SyncEnv -> ByteString -> Bool
 isSimplePlutusScriptHashInWhitelist syncEnv scriptHash = do
   case ioPlutus iopts of
     PlutusEnable -> True
-    PlutusDisable -> False
+    PlutusDisable -> True
```

Comment:
Does this need to change? My comment https://github.com/IntersectMBO/cardano-db-sync/pull/1644#discussion_r1599671182 was about a different function.

Reply:
I don't know anymore 😂 but you're right it's switched back on my working branch 👍